src/lib/components/Communities/CommunityHeader.js

```diff
@@ -34,13 +34,13 @@ class CommunityHeaderComponent extends Component {
               <>
                 <div className="page-subheader-element">
                   <Image
-                    size="mini"
-                    className="community-logo-header"
+                    size="tiny"
```

Comment:
the size of the image was changed many times and finally agreed on `mini`. How does it look like after this change when the image is not "tall" but with square ratio?

Reply:
I would say the first picture is pretty square, you didn't notice it? I changed it because before it could overflow, maybe it looked bigger but i seemed really small to me now.